src/Artifactory/ArtifactoryDownloadClient.ts

```diff
@@ -1,7 +1,14 @@
 import { HttpClient, IRequestParams } from '../HttpClient';
 import { ILogger } from '../../model/';
+import { IChecksumResult } from './Search/ChecksumResult';
+import { IClientResponse } from '../ClientResponse';
+import * as fs from 'fs';
 
 export class ArtifactoryDownloadClient {
+    private static readonly MD5_HEADER: string = 'x-checksum-md5';
+    private static readonly SHA1_HEADER: string = 'x-checksum-sha1';
+    private static readonly SHA256_HEADER: string = 'x-checksum-sha256';
```

Comment:
Just to be consistent with the Go client:
```suggestion
    private static readonly MD5_HEADER: string = 'X-Checksum-Md5';
    private static readonly SHA1_HEADER: string = 'X-Checksum-Sha1';
    private static readonly SHA256_HEADER: string = 'X-Checksum-Sha256';
```